packages/manager/src/dev-tools/FeatureFlagTool.tsx

```diff
@@ -11,6 +11,12 @@ import { setMockFeatureFlags } from 'src/store/mockFeatureFlags';
 import { getStorage, setStorage } from 'src/utilities/storage';
 const MOCK_FEATURE_FLAGS_STORAGE_KEY = 'devTools/mock-feature-flags';
 
+/**
+ * Our flags can be either a boolean or an JSON object
+ * In the case of JSON Objects, the `enabled` key will be used to control flag.
+ * It is required to have the `enabled` key if using a JSON object for on/off Featured flags.
+ * This requirement is both documented here and in our Docs since we don't have a way to enforce types from Launch Darkly objects. .
```

Comment:
```suggestion
 * It is required to have the `enabled` key if using a JSON object for on/off feature flags.
 * This requirement is both documented here and in our Docs since we don't have a way to enforce types from Launch Darkly objects.
```
Very minor nits. Thank you for documenting this here!

Reply:
done!